Content.Server/Containers/EmptyOnMachineDeconstructSystem.cs

```diff
@@ -36,9 +36,11 @@ private void OnDeconstruct(EntityUid uid, EmptyOnMachineDeconstructComponent com
             if (!EntityManager.TryGetComponent<ContainerManagerComponent>(uid, out var mComp))
```

Comment:
```suggestion
            if (!TryComp<ContainerManagerComponent>(uid, out var mComp))
```
since we are doing a cleanup anyways, we can use the shorthands here as well